internal/evaluator/filters_test.go

```
@@ -1796,3 +1796,127 @@ func TestPipelineIntentionWithMultipleValues(t *testing.T) {
+			Message:  "test failure",
+			Metadata: map[string]interface{}{metadataCode: "pkg.rule"},
+		}}
+
+		_, _, exceptions := filter.FilterResults(results, makeRules(), "", "", map[string]bool{}, now)
+
+		require.Len(t, exceptions, 1)
+		assert.NotContains(t, exceptions[0].Metadata, metadataEffectiveOn)
+		assert.NotContains(t, exceptions[0].Metadata, metadataEffectiveUntil)
+	})
+}
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)